wcivf/apps/people/templates/people/person_detail.html

```diff
@@ -26,23 +26,43 @@ <h2 class="ds-candidate-name ds-h3">
           {% endif %}
 
           {% if object.current_or_future_candidacies.all.count > 1 %}
-          {% if object.personpost.party.is_independent %}
-          <h5>{{ object.name }} is an {{ object.personpost.party.party_name }} candidate in the following elections:
-          </h5>
-          {% else %}
-          <h5>{{ object.name }} is a {{ object.personpost.party.party_name }} candidate in the following elections:</h5>
-          {% endif %}
-          {% for candidacy in object.current_or_future_candidacies %}
-          <ul>
-            <li>
-              {{ candidacy.election.name }} for {{ candidacy.post }}
-            </li>
-          </ul>
-          {% endfor %}
+          
+            {% if object.personpost.party.is_joint_party %}
```

Comment:
I might be reading this wrong - but I don't think join parties are quite what you think they are here?

A candidate can stand for one or more elections for a joint party - basically agreeing to campaign/work under the manifestos of more than one party.

That's not the same as that person standing for more than one party in _different candidacies_.

Joint parties are by agreement with the parties in advance, but the case I think you're thinking about in this conditional is where a person has >1 current candidacy and one is for e.g Labour and the other is e.g independent.

Reply:
As discussed in Slack, I had misunderstood joint_parties so thanks for clarifying. I've removed the conditional - making the intro text the same for multiple candidacies regardless of party (although slight edit for 'a' vs 'an' in the case of Independents) and updated the method to account for different ids as described in the previous comment.